xbmc/windowing/XkbcommonKeymap.cpp

```diff
@@ -0,0 +1,351 @@
+/*
```

Comment:
don't bother with mir, maybe it dies soon. for now it should be moved to wayland.